Packs/Core/Playbooks/playbook-Credential_Dumping_using_a_known_tool.yml

```diff
@@ -0,0 +1,628 @@
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: e175d505-aad6-4f06-8898-4b6f2e68782f
+    type: condition
+    task:
+      id: e175d505-aad6-4f06-8898-4b6f2e68782f
+      version: -1
+      name: Analyst approval for isolation
+      description: Analyst approval is required to isolate the endpoint.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "24"
+      "Yes":
+      - "27"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -280,
+          "y": 200
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+        simple: Should perform isolation on the endpoint ${alert.hostname} ?
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+      replyOptions:
+      - "Yes"
+      - "No"
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "30":
+    id: "30"
+    taskid: 02d8f791-2c65-4dab-870c-cd53cf133be9
+    type: regular
+    task:
+      id: 02d8f791-2c65-4dab-870c-cd53cf133be9
+      version: -1
+      name: Get endpoint info by endpoint ID
+      description: Gets a list of endpoints, according to the passed filters. If there are no filters, all endpoints are returned. Filtering by multiple fields is concatenated using the AND condition (OR is not supported). Maximum result set size is 100. Offset is the zero-based number of endpoints from the start of the result set (start by counting from 0).
+      script: '|||core-get-endpoints'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "31"
+    scriptarguments:
+      endpoint_id_list:
+        simple: ${alert.agentid}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 10,
+          "y": -145
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "31":
+    id: "31"
+    taskid: c887cbd2-5a09-4d82-83da-f2df7a9c068f
+    type: condition
+    task:
+      id: c887cbd2-5a09-4d82-83da-f2df7a9c068f
+      version: -1
+      name: Verify endpoint isn't isolated, disconnected, or a server
+      description: Determine whether to isolate the endpoint based on its status, isolation state, and OS type.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "36"
+      Already isolated:
+      - "24"
+      "Yes":
```

Comment:
Maybe 'Isolate' would be more clear instead of 'Yes'

Reply:
Done